BrPersonaNameForm.vue

```diff
@@ -1,17 +1,12 @@
 <template>
   <div class="fit">
-    <q-field
-      class="col-5 q-pr-md"
+    <q-input
+      v-model="name.value"
+      :label="name.label"
       :error="$v.name.$error"
-      helper="Example: Business, Education, Gaming, etc."
```

Comment:
Please add `hint="Example: Business, Education, Gaming, etc."` here to provide a similar result to how we used to have the form with the examples.

Reply:
cool man will do now! thanks!

Reply:
<img width="755" alt="Screen Shot 2019-03-26 at 1 06 38 PM" src="https://user-images.githubusercontent.com/278280/55026240-399aa380-4fc8-11e9-9ec9-0d686a57fdb4.png">

A bit busy @JoshDunnDev

Reply:
I feel like we need to have the Example text to make it clear what the persona name is. We might be able to just remove the placeholder -> `placeholder="Please type a name for your Persona..."` which might provide a less busy look. I don't think the placeholder is quest necessary here since we already have a label.